lgc/elfLinker/RelocHandler.cpp

```diff
@@ -103,6 +110,39 @@ bool RelocHandler::getValue(StringRef name, uint64_t &value) {
     }
   }
 
+  if (name.startswith(reloc::DescriptorUseSpillTable)) {
+    // If the corresponding node is a root node and the type is DescriptorBuffer, use the spill table to get the
+    // descriptor pointer.
+    unsigned descSet = 0;
+    unsigned binding = 0;
+    int typeLetter = 0;
+    StringRef suffix = name.drop_front(strlen(reloc::DescriptorUseSpillTable));
+    if (parseDescSetBinding(suffix, descSet, binding, typeLetter)) {
+      const ResourceNode *outerNode = nullptr;
+      const ResourceNode *node = nullptr;
+      const ResourceNodeType type = LetterToType(typeLetter);
+      std::tie(outerNode, node) = getPipelineState()->findResourceNode(type, descSet, binding);
+      if (node) {
```

Comment:
Again, check if the resource is in the top level by checking if `outerNode == node`.